coverage/generate-index-html.py

```diff
@@ -32,18 +32,36 @@
         main { margin-bottom: 2rem; }
         .table-header,
         .table-row {
+          box-sizing: border-box;
           display: flex;
           width: 100%;
           padding: 2px 10px;
         }
-        .table-header { font-weight: bold; }
+        .table-header { font-weight: bold;}
```

Comment:
Hi @stevemao I need to make another PR now that it has landed or i can just commit to this PR?